web/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.0.0",
   "description": "Curate photos from remote cameras.",
   "scripts": {
-    "build": "npm run build-css && cross-env NODE_ENV=development webpack --progress --hide-modules --watch",
+    "build": "npm run build-css && cross-env NODE_ENV=development webpack --progress --hide-modules",
+    "build-dev": "npm run build-css && cross-env NODE_ENV=development webpack --progress --hide-modules --watch",
```

Comment:
can we just get one more where cross-env NODE_ENV=production?

Reply:
Taken care of in 382350b.